crypto/dilithium/pqcrystals_dilithium_ref_common/packing.c

```diff
@@ -148,37 +157,39 @@ void unpack_sk(uint8_t rho[SEEDBYTES],
 *
 * Description: Bit-pack signature sig = (c, z, h).
 *
-* Arguments:   - uint8_t sig[]: output byte array
+* Arguments:   - ml_dsa_params: parameter struct
+*              - uint8_t sig[]: pointer to output byte array
 *              - const uint8_t *c: pointer to challenge hash length SEEDBYTES
 *              - const polyvecl *z: pointer to vector z
 *              - const polyveck *h: pointer to hint vector h
 **************************************************/
-void pack_sig(uint8_t sig[CRYPTO_BYTES],
-              const uint8_t c[CTILDEBYTES],
+void pack_sig(ml_dsa_params *params,
+              uint8_t *sig,
+              const uint8_t *c,
               const polyvecl *z,
               const polyveck *h)
 {
   unsigned int i, j, k;
 
-  for(i=0; i < CTILDEBYTES; ++i)
+  for(i=0; i < params->c_tilde_bytes; ++i)
     sig[i] = c[i];
-  sig += CTILDEBYTES;
+  sig += params->c_tilde_bytes;
 
-  for(i = 0; i < L; ++i)
-    polyz_pack(sig + i*POLYZ_PACKEDBYTES, &z->vec[i]);
-  sig += L*POLYZ_PACKEDBYTES;
+  for(i = 0; i < params->l; ++i)
+    polyz_pack(params, sig + i * params->poly_z_packed_bytes, &z->vec[i]);
+  sig += params->l*params->poly_z_packed_bytes;
```

Comment:
```suggestion
  sig += params->l * params->poly_z_packed_bytes;
```

Reply:
Thanks, fixed in [ea859d1](https://github.com/aws/aws-lc/pull/1910/commits/ea859d1b4a527b801a531e415122f51475802817).